web/src/lib/components/ui/card/card.svelte

```diff
@@ -0,0 +1,95 @@
+
+  export let icon: Icons;
+
+  export let text: string;
+  export let number: number;
+</script>
+
+<div
+  class="w-1/4 max-w-[300px] h-1/4 rounded-xl bg-white p-4 justify-between flex flex-row top-[50%] transform shadow-custom"
+>
+  <div class="flex flex-col text-primary justify-between">
+    <p class="font-normal text-xl">{text}</p>
+    <p class="text-4xl text-bold mt-6">{number}</p>
+    <p class="invisible">.</p>
+  </div>
+  <div class="flex flex-col justify-between">
+    <div class="bg-primary flex p-2 rounded-lg justify-center">
+      {#if icon === Icons.STUDENT}
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="M12 10.5C13.6569 10.5 15 9.15685 15 7.5C15 5.84315 13.6569 4.5 12 4.5C10.3431 4.5 9 5.84315 9 7.5C9 9.15685 10.3431 10.5 12 10.5Z"
+            stroke="white"
+            stroke-width="1.5"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+          <path
+            d="M19.5 20.4998C19.025 11.1667 4.975 11.1667 4.5 20.4998"
+            stroke="white"
+            stroke-width="1.5"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+        </svg>
+      {:else if icon === Icons.TEACHER}
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="M15.875 6V6.125H16H19C19.7625 6.125 20.4938 6.4279 21.0329 6.96707C21.5721 7.50624 21.875 8.2375 21.875 9V18C21.875 18.7625 21.5721 19.4938 21.0329 20.0329C20.4938 20.5721 19.7625 20.875 19 20.875H5C4.2375 20.875 3.50624 20.5721 2.96707 20.0329C2.4279 19.4938 2.125 18.7625 2.125 18V9C2.125 8.2375 2.4279 7.50624 2.96707 6.96707C3.50624 6.4279 4.2375 6.125 5 6.125H8H8.125V6V5C8.125 4.50272 8.32254 4.02581 8.67418 3.67417C9.02581 3.32254 9.50272 3.125 10 3.125H14C14.4973 3.125 14.9742 3.32254 15.3258 3.67417C15.6775 4.02581 15.875 4.50272 15.875 5V6ZM10 4.875H9.875V5V6V6.125H10H14H14.125V6V5V4.875H14H10ZM4 12.875H3.875V13V18C3.875 18.2984 3.99353 18.5845 4.2045 18.7955C4.41548 19.0065 4.70163 19.125 5 19.125H19C19.2984 19.125 19.5845 19.0065 19.7955 18.7955C20.0065 18.5845 20.125 18.2984 20.125 18V13V12.875H20H16H15.875V13V14C15.875 14.2321 15.7828 14.4546 15.6187 14.6187C15.4546 14.7828 15.2321 14.875 15 14.875C14.7679 14.875 14.5454 14.7828 14.3813 14.6187C14.2172 14.4546 14.125 14.2321 14.125 14V13V12.875H14H10H9.875V13V14C9.875 14.2321 9.78281 14.4546 9.61872 14.6187C9.45462 14.7828 9.23206 14.875 9 14.875C8.76794 14.875 8.54538 14.7828 8.38128 14.6187C8.21719 14.4546 8.125 14.2321 8.125 14V13V12.875H8H4ZM20 11.125H20.125V11V9C20.125 8.70163 20.0065 8.41548 19.7955 8.2045C19.5845 7.99353 19.2984 7.875 19 7.875H5C4.70163 7.875 4.41548 7.99353 4.2045 8.2045C3.99353 8.41548 3.875 8.70163 3.875 9V11V11.125H4H20Z"
+            fill="white"
+            stroke="#2B6E89"
+            stroke-width="0.25"
+          />
+        </svg>
```

Comment:
We shouldn't paste the raw SVG but instead just use an existing icon library. Where do you have those icons from?

Reply:
I got them from a figma plugin. I don't think we can find them in any library.

Reply:
I found some alternatives. Not the best, but they work.